src/Client.php

```diff
@@ -405,6 +405,18 @@ public function send($method, $payload, $path, $params = [], array $headers = []
             return null;
         }
 
+        if (
+            in_array($response->getStatusCode(), [301, 302, 303], true)
+            && $response->hasHeader('Location')
+            && empty((string) $response->getBody())
+        ) {
+            $location = $this->createUri($response->getHeaderLine('Location'));
```

Comment:
What about URI to another host or to another base path?

Reply:
Not supported. I didn't find such cases in Rebilly API. I don't know how to bypass `BaseUri` middleware.

Reply:
At least you can check if and do not send another request. Backend developers won't remember that SDK client has this limitation and if we will add such case the user will hit exceptions.